src/panels/lovelace/editor/view-editor/hui-dialog-edit-view.ts

```diff
@@ -158,10 +158,8 @@ export class HuiDialogEditView extends LitElement {
         .heading=${true}
       >
         <div slot="heading">
-          <h2>
-            ${this._viewConfigTitle}
-          </h2>
-          <paper-tabs
+          <h2>${this._viewConfigTitle}</h2>
+          <ha-tabs
```

Comment:
This should use `mwc-tabs` and not `ha-tabs`

Reply:
Do you mean keep as paper-tabs?

Reply:
I stand corrected